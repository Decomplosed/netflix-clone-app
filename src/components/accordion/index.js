import React from 'react';

export default function Accordion({ children, ...restProps }) {
  return <Container></Container>;
}
