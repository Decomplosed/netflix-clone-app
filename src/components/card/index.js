import React from 'react';

export const FeatureContext = createContext();
