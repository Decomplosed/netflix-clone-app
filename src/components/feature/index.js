import React from 'react';
import { Container, Title, SubTitle } from './styles/feature';

export default function Feature({ children, ...restProps }) {
  return <Container></Container>
}
