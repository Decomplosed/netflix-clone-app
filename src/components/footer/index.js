import React from 'react';

export default function Footer({ children, ...restProps }) {
  return <Container {...restProps}></Container>;
}
