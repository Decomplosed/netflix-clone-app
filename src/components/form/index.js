import React from 'react';
import {
  Container,
  Base,
  Error,
  Title,
  Text,
  TextSmall,
  Link,
  Input,
  Submit,
} from './styles/form';

export default function Form({ children, ...restProps }) {
  return <Container {...restProps}>{children}</Container>;
}

Form.Base = function FormBase({ children, ...restProps }) {
  return <Base {...restProps}>{children}</Base>;
};

Form.Error = function FormError({ children, ...restProps }) {
  return <Error {...restProps}>{children}</Error>;
};

Form.Title = function FormTitle({ children, ...restProps }) {
  return <Title {...restProps}>{children}</Title>;
};
