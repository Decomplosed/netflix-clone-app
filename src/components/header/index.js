import React from 'react';

export default Header({bg = true, children, ...restProps}) {}