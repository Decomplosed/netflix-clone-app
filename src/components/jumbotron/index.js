import React from 'react';

export default Jumbotron = () => {
  return <p>I am jumbotron</p>;
};
