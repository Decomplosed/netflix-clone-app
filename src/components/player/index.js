import React, { useState, useContext, createContext } from 'react';
import ReactDOM from 'react-dom';
