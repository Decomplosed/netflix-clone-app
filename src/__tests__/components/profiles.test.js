import React from 'react';
import { render } from '@testing-library/react';
import { Profiles } from '../../components';
