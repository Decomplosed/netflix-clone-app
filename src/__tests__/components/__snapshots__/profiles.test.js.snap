// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profiles /> Renders the <Profiles /> with populated data 1`] = `
<div
  class="profiles__Container-mfsi0x-0 eElgzR"
>
  <h1
    class="profiles__Title-mfsi0x-1 kjBaoJ"
  >
    Who's watching?
  </h1>
  <ul
    class="profiles__List-mfsi0x-2 cMZVyu"
  >
    <li
      class="profiles__Item-mfsi0x-5 kKqToI"
    >
      <img
        class="profiles__Picture-mfsi0x-4 knQgtl"
        data-testid="profile-picture"
        src="/images/users//images/karl.png.png"
      />
      <p
        class="profiles__Name-mfsi0x-3 gbwdLW"
      >
        Bart
      </p>
    </li>
  </ul>
</div>
`;
